lib/hawkular/inventory/entities.rb

```diff
@@ -1,32 +1,39 @@
 # It contains class definitions that are used by the inventory REST client
 module Hawkular::Inventory
   class Metric
-    # @return [String] Name of the metric
+    # @return [String] Provider Name of the metric
     attr_reader :name
-    # @return [String] Type of the metric
-    attr_reader :type
+    # @return [String] Display Name of the metric
+    attr_reader :display_name
+    # @return [String] Family of the metric (Prometheus family name)
+    attr_reader :family
+    # @return [String] Promql expression for fetching the  time series
+    attr_reader :expression
     # @return [String] Unit of the metric
     attr_reader :unit
+    # @return [String] Expression of the metric (Prometheus expression)
+    attr_reader :expression
+    # @return [Hash<String,String>] Labels of this metric (Prometheus labels)
+    attr_reader :labels
     # @return [Hash<String,String>] Properties of this metric
     attr_reader :properties
 
     def initialize(hash)
-      @name = hash['name']
-      @type = hash['type']
+      @name = hash['displayName']
+      @display_name = hash['displayName']
```

Comment:
name and displayName are the same thing?

Reply:
They are. The metric has only displayName but both fields are required by the code, at least at the moment.